data.sql

```diff
@@ -1,5 +1,3 @@
-/* Populate database with sample data. */
-
 insert into animals values(1,'Agumon','3-FEB-2020',0,1,10.23);
 insert into animals values(2,'Gabumon','15-NOV-2018',2,1,8.0);
 insert into animals values(3,'Pikachu','7-JAN-2021',1,0,15.04);
```

Comment:
- [ ] Kindly note that In PostgreSQL, it's recommended to use the ISO date format (YYYY-MM-DD) for date values. Here I an example.  `2020/2/3`. 
- [ Optional ]  I'd like to suggest that you consider designating the id column as the primary key. By doing so, PostgreSQL will automatically generate unique ID values when you insert records into your database. This can simplify your data insertion process since you won't need to specify the ID value explicitly. It's a handy feature that can make managing your database even smoother. 😊🔑

Reply:
please check. i updated all the above issue. However, date format is not the part of the project core requirement